### DISCUSSION TITLE
1.18.0

### DISCUSSION THREAD

**Discussion by xaoxuu** [xaoxuu/hexo-theme-stellar#208] [Announcements]
- `A` 增加 `tag` 标签
- `A` 时间线增加微博数据支持
- `M` `border` 标签重命名为 `ablock`
- `M` `split` 标签重命名为 `grid`
- `M` `theme.style.color` 配置简化

<hr /><em>This discussion was created from the release <a href='https://github.com/xaoxuu/hexo-theme-stellar/releases/tag/1.18.0'>1.18.0</a>.</em>

**Comment by shaoyaoqian** [comment]
怎么弄一个超级黑的主题色？现在的我设置不来了，我觉得不够黑～

**Reply by xaoxuu** [reply]
你再更新一下，然后这里改成 `100%` 看看够不够黑：

```yaml
style:
  color:
    background: 'hsl(212 16% 100%)'
```

想要纯黑的话，把这个地方改成 `--site-bg: black`：
https://github.com/xaoxuu/hexo-theme-stellar/blob/0a2f0e6087154fbe3b86243544badbee74b804da/source/css/_defines/theme.styl#L48

**Reply by shaoyaoqian** [reply]
好的～ 网易云音乐的上传了
